src/checks.js

```diff
@@ -152,6 +157,10 @@ function checkAttributeValue(name, value) {
 		return CheckFailed(invalidType)
 	} else if (REGEXES.boolean.test(value) && !types.includes("boolean")) {
 		return CheckFailed(invalidType)
+	} else if (REGEXES.function.test(value) && !functions.includes(RegExp.$1)) {
+		return CheckFailed(`Invalid function "${RegExp.$1}".`)
+	} else if (REGEXES.variable.test(value) && !variables.includes(RegExp.$1)) {
```

Comment:
```hcl
response {
  status = 401
```
is reported as 'Invalid variable "401"'.

Reply:
:+1: